code/modules/research/designs/mechfab/prosthetics/internal.dm

```diff
@@ -24,6 +24,11 @@
 	id = "robotic_kidneys"
 	build_path = /obj/item/organ/internal/kidneys
 
+/datum/design/item/mechfab/prosthetic/internal/kidneys
+	name = "Prosthetic Right Kidneys"
```

Comment:
this has the same path as the previous one, which will probably override it. not to mention that the previous one still just says "Kidneys", not "Left Kidney"